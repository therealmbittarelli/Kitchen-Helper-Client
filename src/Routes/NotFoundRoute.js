import React, { Component } from "react";

class NotFoundPage extends Component {
  render() {
    return (
      <div>
        <h2>Oh no!</h2>
        <h3>Error 404: Page Not Found</h3>
      </div>
    );
  }
}

export default NotFoundPage;
