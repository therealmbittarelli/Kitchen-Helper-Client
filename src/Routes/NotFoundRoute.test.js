import React from 'react';
import ReactDOM from 'react-dom';
import NotFoundRoute from './NotFoundRoute';
import { BrowserRouter } from 'react-router-dom';
import renderer from "react-test-renderer";

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<BrowserRouter>
    <NotFoundRoute />
  </BrowserRouter>, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders the UI as expected', () => {
  const tree = renderer.create(<BrowserRouter>
    <NotFoundRoute />
  </BrowserRouter>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});