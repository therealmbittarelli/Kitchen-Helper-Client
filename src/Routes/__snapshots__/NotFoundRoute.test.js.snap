// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div>
  <h2>
    Oh no!
  </h2>
  <h3>
    Error 404: Page Not Found
  </h3>
</div>
`;
